Tests/test_file_webp_animated.py

```diff
@@ -46,14 +46,18 @@ def test_write_animation_L(tmp_path):
             orig.load()
             im.load()
             assert_image_similar(im, orig.convert("RGBA"), 32.9)
+
+            if is_big_endian():
+                webp = parse_version(features.version_module("webp"))
+                if webp < parse_version("1.2.2"):
+                    return
```

Comment:
Perhaps a skip message (and below) instead?

Reply:
Ok, done.